src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/Transformer.kt

```diff
@@ -124,8 +125,8 @@ fun CourtEventChargeResponse.toDpsCharge() = LegacyUpdateCharge(
   offenceEndDate = this.offenceEndDate,
 )
 
-fun OffenderChargeResponse.toDpsMigrationCharge(chargeId: Long, dpsSentence: MigrationCreateSentence?) = MigrationCreateCharge(
-  offenceCode = this.offence.offenceCode,
+fun CourtEventChargeResponse.toDpsMigrationCharge(chargeId: Long, dpsSentence: MigrationCreateSentence?) = MigrationCreateCharge(
+  offenceCode = this.offenderCharge.offence.offenceCode,
```

Comment:
Small but important change - this is taking the data from the appearance version of the charge rather than the underlying charge as each appearance may have different data - for example result code